src/views/Home.vue

```diff
@@ -6,6 +6,11 @@
         $t("linkProjectText")
       }}</a>
     </p>
+
+    <div class="alert alert-info">
```

Comment:
Is the alert dismissable? Not a big deal if it's not

Reply:
Not aware of any easy dismissable extension in WET.

Reply:
Only quick thing I found was https://wet-boew.github.io/wet-boew/demos/collapsible-alerts/collapsible-alerts-en.html